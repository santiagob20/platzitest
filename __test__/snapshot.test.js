import { getCharacter } from '../snapshot'
import rick from '../rick.json'

describe('Instantaneas', () => {

    test('test de un snapshot', () => {
        expect(getCharacter(rick)).toMatchSnapshot()
    })
    test('Siempre fallarÃ¡ la instantanea', () => {
        const user = {
            createdAt: new Date(),
            id: Math.floor(Math.random() * 20)
        }
        expect(user).toMatchSnapshot()
    })
    test('Excepcion del snapshot', () => {
        const user = {
            name: 'Santiago Barrera',
            id: Math.floor(Math.random() * 20)
        }
        expect(user).toMatchSnapshot({
            id: expect.any(Number)
        })
    })
})

