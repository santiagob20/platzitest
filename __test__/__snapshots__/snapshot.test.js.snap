// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instantaneas Excepcion del snapshot 1`] = `
Object {
  "id": Any<Number>,
  "name": "Santiago Barrera",
}
`;

exports[`Instantaneas Siempre fallar√° la instantanea 1`] = `
Object {
  "createdAt": 2021-03-17T01:07:56.112Z,
  "id": 8,
}
`;

exports[`Instantaneas test de un snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanchez",
  "species": "Human",
  "status": "Alive",
}
`;
